### PR TITLE
[13.0][PORT] #622 port from 10.0: force technical user recompute export

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Port #622 from 10.0 to 13.0

### CODE REVIEW

File: shopinvader_search_engine/tests/__init__.py [hunk 1]
```diff
@@ -1 +0,0 @@
-from . import test_delete_product
```
Comment (sebastienbeau) [inline shopinvader_search_engine/tests/__init__.py]: why we remove this import?

Comment (rousseldenis) [reply shopinvader_search_engine/tests/__init__.py]: @HviorForgeFlow

**Comment by rousseldenis** [comment]
base_technical_user is merged in 13 now

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**